scripts/provisioning/resetcmd.py

```diff
@@ -19,9 +19,12 @@
 #
 
 import sys
-
+from cortx.utils.process import SimpleProcess
 from setupcmd import SetupCmd
 
+services_list = ["haproxy", "s3backgroundconsumer", "s3authserver"]
+all_services_instances_list = ["s3backgroundproducer@*", "s3server@*"]
```

Comment:
this service file is not in use yet - so make it  's3backgroundproducer' and move it to 'services_list'.

Reply:
added to list